File: Introibo/Data/Station.swift
import Foundation

// Matches Introibo/Resources/stations.json — 14 stations, ordered.

struct Station: Identifiable, Decodable, Hashable {
    let station: String     // Roman numeral I...XIV
    var title: String
    let latin: String
    var med: String         // Meditation
    let mood: String        // "" | "mood-mother" | "mood-death" | "mood-tomb"
    let stabat_lat: String
    var stabat_eng: String

    var id: String { station }
}

File: android/app/src/main/java/com/lampstandhq/introibo/data/model/Station.kt
package com.lampstandhq.introibo.data.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

/**
 * Matches stations.json — 14 stations, ordered.
 */
@Serializable
data class Station(
    val station: String,     // Roman numeral I...XIV
    val title: String,
    val latin: String,
    val med: String,         // Meditation
    val mood: String,        // "" | "mood-mother" | "mood-death" | "mood-tomb"
    @SerialName("stabat_lat")
    val stabatLat: String,
    @SerialName("stabat_eng")
    val stabatEng: String,
)
